region/wreckedship/main/Wrecked Ship Main Shaft.json

```
@@ -861,12 +861,69 @@
         {"obstaclesNotCleared": ["B"]},
         {"canShineCharge": {"usedTiles": 13, "openEnd": 1}}
       ],
+      "clearsObstacles": ["C"],
       "unlocksDoors": [
         {"nodeId": 3, "types": ["missiles", "super"], "requires": []},
         {"nodeId": 3, "types": ["powerbomb"], "requires": ["never"]}
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [3, 8],
+      "name": "Atomic Ice Moonfall Screw Clip",
+      "requires": [
+        {"notable": "Atomic Ice Moonfall Screw Clip"},
+        "f_DefeatedPhantoon",
+        {"enemyDamage": {"enemy": "Atomic", "type": "contact", "hits": 1}},
+        "canUseIFrames",
+        "canTrickyUseFrozenEnemies",
+        "canEnemyStuckMoonfall",
+        "ScrewAttack",
+        "Morph"
+      ],
+      "clearsObstacles": ["C"],
+      "note": [
+        "Freeze an Atomic near the bottom of the stairs above the bomb block, so that it is barely off-camera when Samus drops down.",
+        "Below, prepare a second Atomic to be frozen.",
+        "As it approaches, jump up to cause the upper Atomic to come on-camera and descend.",
+        "Take a hit from the lower Atomic, and freeze it about 1.5 tiles above the floor;",
+        "use the i-frames to jump on top of the lower Atomic and freeze the upper Atomic while it is inside Samus' head.",
+        "Immediately moonfall to the left.",
+        "If successful, Samus will clip two tiles into the floor.",
+        "Turn to the right, and perform a turnaround spin jump to the left to break the bomb block using Screw Attack.",
+        "If Samus clips through the Atomics but does not clip into the floor, it means the bottom Atomic was frozen too low.",
+        "If Samus clips only one tile into the floor, it means the bottom Atomic was frozen too high."
+      ]
+    },
+    {
+      "link": [3, 8],
+      "name": "Covern Wall Ice Screw Clip",
+      "requires": [
+        {"notable": "Covern Wall Ice Screw Clip"},
+        {"not": "f_DefeatedPhantoon"},
+        {"or": [
+          "canRiskPermanentLossOfAccess",
+          {"and": [
+            {"notable": "Atomic Ice Moonfall Screw Clip"},
+            {"enemyDamage": {"enemy": "Atomic", "type": "contact", "hits": 1}}
+          ]}
+        ]},
```

```suggestion
        ]},
        "canMidairWiggle",
```
Quickly turning back around in order to activate screw attack is a wiggle.